src/views/workbench/workbench.tsx

```diff
@@ -77,12 +77,23 @@ const styles: StyleRulesCallback<CssRules> = (theme: ArvadosTheme) => ({
 
 type WorkbenchPanelProps = WithStyles<CssRules>;
 
+const defaultSplitterSize = 90;
+
+const getSecondaryInitialSize = () => {
+    const splitterSize = localStorage.getItem('splitterSize');
+    return splitterSize ? Number(splitterSize) : defaultSplitterSize;
+};
+
+const onPaneSizeChange = (size: number) => localStorage.setItem('splitterSize', size.toString());
```

Comment:
`on...` it's good for callback properties names but here I would simply give `setSplitterSize`